tidb-cloud/select-cluster-tier.md

```diff
@@ -17,7 +17,20 @@ TiDB Cloud provides the following two options of cluster tiers. Before creating
 
 The TiDB Cloud Serverless Tier (previously called Developer Tier) is a fully managed service of TiDB. It's still in beta and should not be used in production. However, you can use Serverless Tier clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for your datasets.
 
-For each TiDB Cloud account, you can create a maximum of five complimentary Serverless Tier clusters to use during the beta phase.
+### Usage quota
+
+For each organization in TiDB Cloud, you can create a maximum of five Serverless Tier clusters by default. To create more Serverless Tier clusters, you need to add a credit card and set a [spend limit](/tidb-cloud/tidb-cloud-glossary.md#spend-limit) for the usage.
+
+For each Serverless Tier cluster, TiDB Cloud provides a free usage quota as follows:
```

Comment:
Only maximum of five complimentary Serverless Tier clusters  have free usage quota.